.github/workflows/actions/deploy_v2/action.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy to AKS
+description: deploys application
+inputs:
+  environment:
+    description: Environment to deploy to
+    required: true
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  azure-credentials:
+    description: Credentials for azure
+    required: true
+outputs:
+  deploy-url:
+    value: ${{ steps.set_env_var.outputs.deploy_url }}
+runs:
+  using: composite
+  steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        tf_vars_file=terraform/aks/config/${{ inputs.environment }}.tfvars.json
+        terraform_version=$(awk '/{/{f=/^terraform/;next}f' terraform/aks/provider.tf | grep -o [0-9\.]*)
+        echo "cluster=$(jq -r '.cluster' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "aks_app_environment=$(jq -r '.environment' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "TERRAFORM_VERSION=$terraform_version" >> $GITHUB_ENV
+        echo "namespace=$(jq -r '.namespace' ${tf_vars_file})" >> $GITHUB_ENV
+        if [[ ${{ inputs.environment }} == 'development_aks' ]]; then
+
+          echo "deploy_url=https://getintoteachingapi.${{ env.aks_app_environment }}.test.teacherservices.cloud" >> $GITHUB_OUTPUT
```

Comment:
shouldn't it be getintoteaching-${{env.aks_app_environment?
plus maybe something like
```
if [[ $cluster == 'production' ]]; then
              echo "deploy_url=https://getintoteachingapi-${{ env.aks_app_environment }}.teacherservices.cloud" >> $GITHUB_OUTPUT
            else
echo "deploy_url=https://getintoteachingapi-${{ env.aks_app_environment }}.${cluster}.teacherservices.cloud" >> $GITHUB_OUTPUT
```